docs/getting-started/user-guides/how-to-vote.md

```diff
@@ -0,0 +1,61 @@
+# **How To Vote**
+
+This guide assumes you already have an active wallet and have purchased tickets. If not, please follow the [Voting Preperation](https://docs.decred.org/getting-started/user-guides/agenda-voting/#voting-preparation) guide.
+
+The choice a ticket votes with depends on your vote preference at the time the ticket is chosen, not when it is bought. So you can set your choice at any time within the voting window and all future tickets will vote accordingly.
+
+## **Solo Voting**
+
+##I just want the commands!##
+
+###**YES**###
+`dcrctl --wallet setvotechoice lnfeatures yes`
+
+###**NO**###
+`dcrctl --wallet setvotechoice lnfeatures no`
+
+----------------
+
+Through the command-line, you'll want to familiarize yourself with the `dcrctl --wallet getvotechoices` and `dcrctl --wallet setvotechoice "agendaid" "choiceid"` commands.
```

Comment:
This should be `command line`.  You would only hyphenate it when it's used as an adjective such as `command-line options`.